pkg/universe.dagger.io/docker/build.cue

```diff
@@ -94,7 +95,9 @@ import (
 		"source":     source
 		"auth":       auth
 		"dockerfile": dockerfile
-		"platforms":  platforms
+		if platform != _|_ {
+			"platform": platform
+		}
```

Comment:
Platform is also optional in `core.#Dockerfile` so you should not need that if statement